### PR TITLE
[stari_task0]: HelloWorld pr.

### PR DESCRIPTION
**Pull request by Bruce-Jay**
### What?

- brief: C++ run test and pr practice
- author: Bruce-Jay

### What changes are included in this PR?

created folder stari in stl/ and created hello.cpp file, compiled and executed it.

### Are these changes tested?

image

### CODE REVIEW

File: stl/stari/hello.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main(void)
+{
```
Comment (Light-City) [inline stl/stari/hello.cpp:4]: int main() {

}

**Comment by Light-City** [comment]
1.please remove bin file , hello.

2.The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

**Comment by Bruce-Jay** [comment]
Thank you! And would you have a view of my new commits?

**Comment by Bruce-Jay** [comment]
I hope that it's OK now

**Comment by Light-City** [comment]
1..DS_Store should remove

2.YirenLin -> yirenlin

lowercase underscore style.

**Comment by Bruce-Jay** [comment]
Thank you for your patience!

File: stl/stari/yirenlin/src/hello.cpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main(void) {
+    std::cout << "Hello World" << std::endl;
```
Comment (martinx) [inline stl/stari/yirenlin/src/hello.cpp:4]: Indent two spaces

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**